app/static/css/base.css

```diff
@@ -31,6 +31,11 @@
 
 .footerlink {
   font-weight: bold;
+
+}
+
+.size{
```

Comment:
The class name "size" is pretty unindicative. A slightly more obvious name would be helpful